lib/util.js

```diff
@@ -87,7 +87,7 @@ util.catch = function(caught) {
     case 404:
       throw new errors.ResourceNotFoundError(options.uri);
     case 409:
-      throw new errors.VehicleStateError(options.uri);
+      throw new errors.VehicleStateError(options.uri, body.code);
```

Comment:
this should be modified to pass `body.message` instead of the URL

Reply:
done!